src/sdk/pynni/nni/compression/torch/utils/counter.py

```diff
@@ -4,15 +4,14 @@
 import torch
 import torch.nn as nn
 from nni.compression.torch.compressor import PrunerModuleWrapper
-
 try:
     from thop import profile
 except Exception as e:
     print('thop is not found, please install the python package: thop')
     raise
 
-
-def count_flops_params(model: nn.Module, input_size, verbose=True):
+from thop.vision.basic_hooks import zero_ops
+def count_flops_params(model: nn.Module, input_size, custom_ops={}, verbose=True):
```

Comment:
`{}` is evaluated on when `count_flops_params` get defined, so it will always be the same object for each call.